src/Avalonia.FreeDesktop/DBusMenuExporter.cs

```diff
@@ -37,30 +46,48 @@ class DBusMenuExporterImpl : ITopLevelNativeMenuExporter, IDBusMenu, IDisposable
             private readonly HashSet<NativeMenu> _menus = new HashSet<NativeMenu>();
             private bool _resetQueued;
             private int _nextId = 1;
+            private bool AppMenu = true;
```

Comment:
can we remove that variable and in Init method write 
```if(ObjectPath!=GenerateDBusMenuObjPath)```

Reply:
If not i will suggest making it readonly and writing it in _camelCase

Reply:
renamed.